### PR TITLE
ui: Remove legacy ACLs

### PR DESCRIPTION
**Pull request by johncowen**
This PR removes all leftover old style "legacy ACLs" from the UI. It's 'kind of' a continuation of https://github.com/hashicorp/consul/pull/11017, in which we only removed things that were complicating adding support for Admin Partitions in the UI, ideally we would have done this PR before that one so we've decided it would be best to finish up removing legacy ACL support completely before moving on with finishing up Admin Partition support.

Just to be clear, this PR is not blocked on any backend support/removal of legacy ACLs.

Most changes are deletions, pretty much all that remains for some redirecting for where the old style ACLs used to live to take you straight to the ACLs Tokens page. There're probably still some CSS code that we no longer need here, but I'll do a larger CSS cleanup later as part of a separate iterative PR.

I'll try add some inline self-review a little later.

### CODE REVIEW

File: ui/packages/consul-ui/app/services/repository/token.js [hunk 1]
```diff
@@ -53,21 +45,14 @@ export default class TokenService extends RepositoryService {
 
   @dataSource('/:partition/:ns/:dc/token/self/:secret')
   self(params) {
-    // TODO: Does this need ns passing through?
+    // This request does not need ns or partition passing through as its
+    // inferred from the token itself.
```
Comment (johncowen) [inline ui/packages/consul-ui/app/services/repository/token.js:49]: I checked in with the team about this and confirmed that it does not need `ns` or `partition` passing, so I fixed this up while I was here and added some more information here to help future me when I next come here

**Review by johncowen — COMMENTED** [review]
Added inline notes

**Review by kaxcode — APPROVED** [review]
👏

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/453704.

**Pull request merged**